tools/topology/m4/dcblock_coef_default.m4

```diff
@@ -0,0 +1,24 @@
+CONTROLBYTES_PRIV(DCBLOCK_priv,
```

Comment:
Can we have a comment on the tool used to generate this byte sequence.

Reply:
@lgirdwood I added the scripts to generate the byte sequences. Could you please review them?